Sources/Core/Network/Client+Async Request.swift

```diff
@@ -0,0 +1,137 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import Alamofire
+import CocoaLumberjack
+
+@available(iOS 13, *)
+public extension Client {
+	/// Shortcut method for building the request and loading the response (ignoring data).
+	///
+	/// - parameter request: The router request type
+	/// - parameter automaticallyCancelling: Bool determining whether or not the request should be cancelled when the enclosing async context is cancelled.
+	///
+	/// - returns: the result
+	func requestVoid(
+		_ request: RouterType,
+		automaticallyCancelling: Bool = false
+	) async -> Result<Void, Error> {
+		let task = self.request(request).serializingResponse(using: PassthroughResponseSerializer(), automaticallyCancelling: automaticallyCancelling)
+		return Self.transform(response: await task.response)
+	}
+
+	/// Shortcut method for building the request and loading the data.
+	///
+	/// - parameter request: The router request type
+	/// - parameter automaticallyCancelling: Bool determining whether or not the request should be cancelled when the enclosing async context is cancelled.
+	///
+	/// - returns: the result
+	func requestData(
+		_ request: RouterType,
+		automaticallyCancelling: Bool = false
+	) async -> Result<Data, Error> {
+		let task = self.request(request).serializingData(automaticallyCancelling: automaticallyCancelling)
+		return Self.transform(response: await task.response)
+	}
+
+	/// Shortcut method for building the request and parsing the JSON.
+	///
+	/// - parameter request: The router request type
+	/// - parameter options: The JSON serialization reading options. Defaults to `.allowFragments`.
+	/// - parameter automaticallyCancelling: Bool determining whether or not the request should be cancelled when the enclosing async context is cancelled.
+	///
+	/// - returns: the result
+	@available(*, deprecated, message: "Will be removed in Alamofire 6. Use `Decodable` instead.")
+	func requestJSON(
+		_ request: RouterType,
+		options: JSONSerialization.ReadingOptions = .allowFragments,
+		automaticallyCancelling: Bool = false
+	) async -> Result<Any, Error> {
+		let task = self.request(request).serializingResponse(using: JSONResponseSerializer(options: options), automaticallyCancelling: automaticallyCancelling)
+		return Self.transform(response: await task.response)
+	}
```

Comment:
I'd just remove this. Does anyone even use the `requestJSON` function (instead of the decodable one)?

Reply:
I still use it in cases for more optimized pagination imports with core data

Reply:
But you do realize it'll be removed in AlamoFire 6 right?

How do you use it that is faster than paginated imports? Because if they're paginated, it won't be large amounts of data… Or do you mean by not using Groot & directly importing yourself?

Reply:
Note: just looking for input, this convo isn't blocking for merge.